IOS_SYMBOLICATION.md

```diff
@@ -0,0 +1,76 @@
+# Symbolicating iOS crash reports
+
+Debugging an iOS application crash sometimes involves analyzing crash reports.
+More info about crash reports can be found
+[in the official documentation](https://developer.apple.com/library/archive/technotes/tn2151/_index.html).
+
+Crash reports generally require symbolication to become properly readable:
+symbolication turns raw hex addresses into human-readable source locations.
+The document below describes some specific details of symbolicating crash reports
+from iOS applications using Kotlin.
```

Comment:
iOS/watchOS/tvOS I guess.

Reply:
A bit cumbersome, considering the fact that this should be used in document title too. Do you have other suggestions?

Reply:
AppStore applications?

Reply:
Not exactly, since the symbolication is applicable to applications that are not distributed through App Store.